:up: Update ES image to be one with latest mappings and data

st3v3nhunt opened the pull request:
Use the latest image for [`profiles-db-elastic`](https://hub.docker.com/r/nhsuk/profiles-db-elastic/tags/) i.e. `0.4.1-20170930`.
This image has the latest set of mappings and data added to the ES instance via the etl combiner and updater jobs. The changes required have been to update the data in accordance with that in the image.

st3v3nhunt commented on test/integration/resultsPage.js:
```diff
@@ -140,46 +140,31 @@ describe('Results page', () => {
   });
 
   describe('Surgeries with booking system which can not be linked to', () => {
-    describe('when the surgery has a website', () => {
```
I can't see another surgery where this condition is true so I've removed the test.

markysoft replied:
How about 'Sunlight Centre Surgery'?

st3v3nhunt replied:
Great suggestion. How did you find it?

st3v3nhunt commented:
Used the suggestion of `Sunlight Centre Surgery` to get all tests covered.
Also updated the packages and removed the snyk policy file as it is no longer needed, as in there are no vulnerabilities needing to be ignored.
